docs/content/preview/drivers-orms/java/yugabyte-jdbc.md

```diff
@@ -57,6 +57,8 @@ type: docs
 
 For Java applications, the JDBC driver provides database connectivity through the standard JDBC application program interface (APIs) available on the Java platform.
 
+The driver package includes a `YBClusterAwareDataSource` class that uses one initial contact point for the YugabyteDB cluster as a means of discovering all the nodes and, if required, refreshing the list of live endpoints with every new connection attempt. The refresh is triggered if stale information (by default, older than 5 minutes) is discovered.
```

Comment:
I see that in the code example below, we really don't use this `YBClusterAwareDataSource` class or the Hikari APIs. We can remove them from this page.

The JDBC application can be written either with the `DriverManager.getConnection()` API or the `DataSource` API. On this page, we are using the former way.

Reply:
Right, it is the Reference page example that uses YBClusterAwareDataSource; I will move it back there. Generally though, this information isn't provided in a single location which could be inconvenient.